### ISSUE TITLE
Cannot create the window

### ISSUE THREAD

**Issue by clarkebowers** [microsoft/PTVS#5814]
**Describe the bug**
[Window Title]
devenv.exe

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[^] Hide details  [Close]

[Expanded Information]
```
Build: 16.3.19252.1

System.Runtime.InteropServices.COMException (0x80131604): Cannot create the window.
   at Microsoft.Internal.VisualStudio.PlatformUI.WindowHelper.GetOrCreateFrameworkElement[T](IVsUIWpfElement element)
   at Microsoft.Internal.VisualStudio.PlatformUI.WindowHelper.GetFrameworkElementFromUIElement(IVsUIElement element)
   at Microsoft.VisualStudio.PlatformUI.InfoBarHostControl.InfoBarFrameworkElementCollection.AdaptItem(IVsUIElement item)
   at Microsoft.Internal.VisualStudio.PlatformUI.CollectionAdapter`2.InsertSourceItem(Int32 index, TSource item)
   at Microsoft.Internal.VisualStudio.PlatformUI.CollectionAdapter`2.OnInnerCollectionChanged(IEnumerable innerCollection, NotifyCollectionChangedEventArgs e)
   at Microsoft.Internal.VisualStudio.PlatformUI.CollectionAdapter`2.System.Windows.IWeakEventListener.ReceiveWeakEvent(Type managerType, Object sender, EventArgs e)
   at System.Windows.WeakEventManager.ListenerList.DeliverEvent(Listener& listener, Object sender, EventArgs args, Type managerType)
   at System.Windows.WeakEventManager.ListenerList`1.DeliverEvent(Object sender, EventArgs e, Type managerType)
   at System.Windows.WeakEventManager.DeliverEvent(Object sender, EventArgs args)
   at System.Collections.Specialized.CollectionChangedEventManager.OnCollectionChanged(Object sender, NotifyCollectionChangedEventArgs args)
   at System.Collections.ObjectModel.ObservableCollection`1.OnCollectionChanged(NotifyCollectionChangedEventArgs e)
   at System.Collections.ObjectModel.ObservableCollection`1.InsertItem(Int32 index, T item)
   at System.Collections.ObjectModel.Collection`1.Add(T item)
   at Microsoft.VisualStudio.PlatformUI.InfoBarHostControl.AddInfoBar(IVsUIElement uiElement)
   at Microsoft.PythonTools.Project.PythonInfoBar.FinishCreate()
   at Microsoft.PythonTools.Project.TestFrameworkInfoBar.ShowInfoBar(String infoBarMessage, List`1 acceptActionItems)
   at Microsoft.PythonTools.Project.TestFrameworkInfoBar.<CheckAsync>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Project.TestFrameworkWorkspaceInfoBar.<CheckAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.WorkspaceInfoBarManager.<TriggerInfoBarsAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Infrastructure.VSTaskExtensions.<HandleAllExceptions>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
```

**Steps to Reproduce**
1. Open solution in VS2019
2. Boom
3. No step three

**Expected behavior**
No boom.

**Additional context and screenshots**
[Window Title]
devenv.exe

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[^] Hide details  [Close]

[Expanded Information]
```
Build: 16.3.19252.1

System.Runtime.InteropServices.COMException (0x80131604): Cannot create the window.
   at Microsoft.Internal.VisualStudio.PlatformUI.WindowHelper.GetOrCreateFrameworkElement[T](IVsUIWpfElement element)
   at Microsoft.Internal.VisualStudio.PlatformUI.WindowHelper.GetFrameworkElementFromUIElement(IVsUIElement element)
   at Microsoft.VisualStudio.PlatformUI.InfoBarHostControl.InfoBarFrameworkElementCollection.AdaptItem(IVsUIElement item)
   at Microsoft.Internal.VisualStudio.PlatformUI.CollectionAdapter`2.InsertSourceItem(Int32 index, TSource item)
   at Microsoft.Internal.VisualStudio.PlatformUI.CollectionAdapter`2.OnInnerCollectionChanged(IEnumerable innerCollection, NotifyCollectionChangedEventArgs e)
   at Microsoft.Internal.VisualStudio.PlatformUI.CollectionAdapter`2.System.Windows.IWeakEventListener.ReceiveWeakEvent(Type managerType, Object sender, EventArgs e)
   at System.Windows.WeakEventManager.ListenerList.DeliverEvent(Listener& listener, Object sender, EventArgs args, Type managerType)
   at System.Windows.WeakEventManager.ListenerList`1.DeliverEvent(Object sender, EventArgs e, Type managerType)
   at System.Windows.WeakEventManager.DeliverEvent(Object sender, EventArgs args)
   at System.Collections.Specialized.CollectionChangedEventManager.OnCollectionChanged(Object sender, NotifyCollectionChangedEventArgs args)
   at System.Collections.ObjectModel.ObservableCollection`1.OnCollectionChanged(NotifyCollectionChangedEventArgs e)
   at System.Collections.ObjectModel.ObservableCollection`1.InsertItem(Int32 index, T item)
   at System.Collections.ObjectModel.Collection`1.Add(T item)
   at Microsoft.VisualStudio.PlatformUI.InfoBarHostControl.AddInfoBar(IVsUIElement uiElement)
   at Microsoft.PythonTools.Project.PythonInfoBar.FinishCreate()
   at Microsoft.PythonTools.Project.TestFrameworkInfoBar.ShowInfoBar(String infoBarMessage, List`1 acceptActionItems)
   at Microsoft.PythonTools.Project.TestFrameworkInfoBar.<CheckAsync>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Project.TestFrameworkWorkspaceInfoBar.<CheckAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.WorkspaceInfoBarManager.<TriggerInfoBarsAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Infrastructure.VSTaskExtensions.<HandleAllExceptions>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
```

**Sample Code**
```
XXX
```

**Configuration information (If you are providing a diagnostics file (see below), skip this section)**
VS Version: [Can be found under "Help->About Microsoft Visual Studio"]
PTVS version: [Can be found under "Help->About Microsoft Visual Studio->Installed products"]
Python version:
Debugger Type (if applicable): [New debugger or legacy debugger?]

**Diagnostics file**
The diagnostics file contains information such as the Python version, environments, loaded assemblies, configuration options, etc. This file makes it easier for us to reproduce and fix bugs, so we would really appreciate it if you could provide it to us. This file can be generated by going to Tools->Options->Python->Diagnostics.
You can attach the file to this issue or email it to us at ptvshelp@microsoft.com. If you email it, please add a link to this issue.
```
XXX
```

**Comment by huguesv** [comment]
Thank you

**Comment by huguesv** [comment]
In case this is happening every time and you need a workaround, you can try unchecking the following options:
![image](https://user-images.githubusercontent.com/1696845/66781742-f2eb9880-ee88-11e9-9917-f6859c3e281e.png)

**Comment by clarkebowers** [comment]
thanks

**Comment by judej** [comment]
The cause of the crash should be fixed in VS 16.4. Please reopen if you continue to see it
